### PR TITLE
Bump version

### LINKED ISSUE #185: Dependency hell
If used with `wellcomeml=2.0.0`, because of some combination of package upgrades, `pip install ` doesn't work anymore (see below)

Screenshot 2022-02-08 at 10 57 40

### PR DESCRIPTION
**Pull request by aCampello**
## Description

I need a new release of the model, because the FortyTwo pipeline was broken due to #185 (which was fixed now)

I can't update dvc because AWS doesn't have enough resources to run it today.

## Checklist

- [ ] Linked to Notion or GitHub issue
- [ ] Added tests
- [ ] Updated README
- [ ] DVC up to date

## Release checklist

- [ ] DVC repro up to date
- [x] Models synced in S3
- [x] Version updated
- [x] CHANGELOG.md updated
- [x] Model and package version aligned

To release:

* `make build`
* `make deploy`

### CODE REVIEW

**Comment by aCampello** [comment]
Update: dvc repro running now

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@
 PRIVATE_PROJECT_BUCKET := $(PROJECTS_BUCKET)/$(PROJECT_NAME)
 PUBLIC_PROJECT_BUCKET := datalabs-public/$(PROJECT_NAME)
 
-PACKAGE_VERSION := $(shell venv/bin/python -c "import grants_tagger;print(grants_tagger.__version__.__version__)")
+PACKAGE_VERSION := $(shell venv/bin/python -c "from grants_tagger.__version__ import __version__; print(__version__)")
```
Comment (nsorros) [inline Makefile:7]: I have been meaning to fix this for some time 😂

**Review by nsorros — APPROVED** [review]
DVC seems updated is it not? In any case I will probably run dvc when retraining BertMesh for hugging face.

**Comment by aCampello** [comment]
Yes, dvc finished overnight 🎉

**Pull request merged**